test/gtest/cba_find2.hpp

```diff
@@ -0,0 +1,187 @@
+#include "conv_common.hpp"
+
+#include "conv_test_base.hpp"
+#include "conv_tensor_gen.hpp"
+
+template <typename T = float>
+struct ConvBiasActivInferFind2Test
+    : public ::testing::TestWithParam<
+          std::tuple<miopenActivationMode_t, ConvTestCase, miopenTensorLayout_t>>,
+      ConvFwdSolverTestBase<T>
+{
+protected:
+    ConvTestCase conv_config;
+    miopen::ActivationDescriptor activ_desc;
+    tensor<T> bias;
+    miopen::Allocator::ManageDataPtr bias_dev;
+    bool test_skipped = false;
+    miopenActivationMode_t activ_mode;
+    /*
+    miopen::FusionPlanDescriptor fusePlanDesc;
+    */
+    miopen::OperatorArgs params;
+    const float alpha       = static_cast<float>(1.0f);
+    const float beta        = static_cast<float>(0);
+    const float activ_alpha = static_cast<double>(0.5f);
+    const float activ_beta  = static_cast<double>(0.5f);
+    const float activ_gamma = static_cast<double>(0.5f);
+    miopenTensorLayout_t tensor_layout;
+
+    miopen::FusedProblem fused_problem;
+    miopen::AnyInvokeParams invoke_params;
+
+    using cfsb = ConvFwdSolverTestBase<T>;
+
+    void SetUp() override
+    {
+        test_skipped                                     = false;
+        std::tie(activ_mode, conv_config, tensor_layout) = GetParam();
+
+        cfsb::SetUpImpl(conv_config, tensor_layout);
+        activ_desc = {activ_mode, activ_alpha, activ_beta, activ_gamma};
+        bias       = tensor<T>{1, static_cast<size_t>(conv_config.k), 1, 1};
+        bias.generate(tensor_elem_gen_integer{3});
+        std::fill(
+            cfsb::output.begin(), cfsb::output.end(), std::numeric_limits<double>::quiet_NaN());
+        bias_dev = get_handle().Write(bias.data);
+
+        // Setup the fusion problem
+        fused_problem = miopen::FusedProblem{{
+            MakeConvProblem(),
+            // MakeBiasProblem(),
+            MakeActivationProblem(),
+        }};
+
+        fused_problem.PropagateDescriptors();
+        ASSERT_NO_THROW(invoke_params = MakeInvokeParams());
+
+        /*
+        fusePlanDesc = miopen::FusionPlanDescriptor(miopenVerticalFusion, cfsb::input.desc);
+        auto convOp =
+            std::make_shared<miopen::ConvForwardOpDescriptor>(cfsb::conv_desc, cfsb::weights.desc);
+        auto biasOp  = std::make_shared<miopen::BiasFusionOpDescriptor>(bias.desc);
+        auto activOp = std::make_shared<miopen::ActivFwdFusionOpDescriptor>(activ_desc.GetMode());
+        EXPECT_EQ(fusePlanDesc.AddOp(convOp), miopenStatusSuccess);
+        convOp->SetArgs(params, &alpha, &beta, cfsb::wei_dev.get());
+        EXPECT_EQ(fusePlanDesc.AddOp(biasOp), miopenStatusSuccess);
+        biasOp->SetArgs(params, &alpha, &beta, bias_dev.get());
+        EXPECT_EQ(fusePlanDesc.AddOp(activOp), miopenStatusSuccess);
+        activOp->SetArgs(params, &alpha, &beta, activ_alpha, activ_beta, activ_gamma);
+         */
+    }
+
+    void TearDown() override
+    {
+        if(test_skipped || checks_ran)
```

Comment:
If the purpose of `checks_ran` is to not recompute the host side result, we can still skip the CPU computation but the verification should still happen.

Reply:
You are right, will remove it. I have added a check to skip repeating cpu computation, but forgot to remove this one.